lib/src/components/EditorFieldTemplate/index.tsx

```diff
@@ -54,27 +47,37 @@ export default function EditorFieldTemplate({ id, schema, children }: FieldTempl
       {children}
       {!isSection && (!isRoot || verna.selector) && (
         <>
-          <button onClick={() => setIsEditing(!isEditing)}>
-            <FormattedMessage {...messages.parameters} />
-          </button>
+          <div style={{ display: 'flex', gap: '5px' }}>
+            <button onClick={() => setIsEditing(!isEditing)}>
+              <FormattedMessage {...messages.parameters} />
+            </button>
+            {canAddField && (
+              <button
+                onClick={() => removeWidget(verna, id)}
+                style={{ height: '24px', width: '24px' }}
```

Comment:
```suggestion
                style={{ height: '24px', width: '24px' }}
```
as it's used twice, we could use a const to define the styleButton and reuse it for the other Button